src/components/Bootstrap.jsx

```diff
@@ -63,17 +64,15 @@ function BootstrapField({
                 style={{
                     position: 'relative',
                     height: `${styles.field_height}${styles.field_height_unit}`,
-                    width: `${styles.field_width}${styles.field_width_unit}`,
-                    maxWidth: '100%'
+                    width: '100%'
```

Comment:
can you check that both px and percent width units look good for these fields with the updated css? i remember there was a specific reason why i replicated the width definition at each level but i can't remember right now.